tensorflow_lite_support/python/task/audio/core/tensor_audio.py

```diff
@@ -0,0 +1,132 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""TensorAudio class."""
+
+import numpy as np
+
+from tensorflow_lite_support.python.task.audio.core.pybinds import _pywrap_audio_buffer
+from tensorflow_lite_support.python.task.audio.core import audio_record
+
+_CppAudioFormat = _pywrap_audio_buffer.AudioFormat
+_LoadAudioBufferFromFile = _pywrap_audio_buffer.LoadAudioBufferFromFile
+
+
+class TensorAudio(object):
+  """A wrapper class to store the input audio."""
+
+  def __init__(self,
+               audio_format: _CppAudioFormat,
+               buffer_size: int) -> None:
+    """Initializes the `TensorAudio` object.
+
+    Args:
+      audio_format: format of the audio.
+      buffer_size: buffer size of the audio.
+    """
+    self._format = audio_format
+    self._buffer_size = buffer_size
+    self._buffer = np.zeros(
+        [self._buffer_size, self._format.channels], dtype=np.float32)
+
+  def clear(self):
+    """Clear the internal buffer and fill it with zeros."""
+    self._buffer.fill(0)
+
+  @classmethod
+  def create_from_wav_file(cls,
+                           file_name: str,
+                           buffer_size: int) -> "TensorAudio":
```

Comment:
Is this value required? Can we have a default value for `buffer_size`?

Reply:
This is required because each audio model has a fixed input size, hence they can only take a TensorAudio of a fixed size as the input.

@kinaryml Let's clarify better in the docstring that this method only loads buffer_size samples at the beginning of the file, same as the docstring of the underlying C++ implementation.

Reply:
@khanhlvg Fixed it.

Reply:
Let's change it a bit for better clarification:
* Rename buffer_size to sample_count.
* sample_count: The number of samples to read from the WAV file. This value should match with the input size of the TensorFlow Lite audio model that will consume the created TensorAudio object. If the WAV file contains more samples than `sample_coubt`, only the samples at the beginning of the WAV file will be loaded.